charts/pgbouncer/templates/_helpers.yaml

```diff
@@ -0,0 +1,70 @@
+{{- define "pgbouncer.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "pgbouncer.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+
+{{ define "pgbouncer_image" -}}
+{{ printf "%s:%s" .Values.image.repository .Values.image.tag }}
+{{- end }}
+
+{{ define "pgbouncer_host" -}}
+{{ printf "%s-pgbouncer" .Release.Name }}
+{{- end }}
+
+{{ define "pgbouncer.port" -}}
+{{- .Values.servicePort }}
+{{- end }}
+
+
+{{ define "pgbouncer_config_secret" -}}
+{{ printf "%s-pgbouncer-config" .Release.Name }}
+{{- end }}
+
+{{ define "pgbouncer_stats_secret" -}}
+{{ .Release.Name }}-pgbouncer-stats
+{{- end }}
+
+{{- define "pgbouncer_environment" }}
+- name: DB_USER
+  value: postgres
+- name: DB_PASSWORD
+  value: postgres
+# - name: DB_NAME
+#   value: postgres
```

Comment:
Seems like if we're leaving all the other defaults in here, like DB_USER = postgres, then we should uncomment this too.

Reply:
unfortunately it will not work, cause it generates concrete DB_NAME https://github.com/astronomer/ap-vendor/blob/main/pgbouncer-krb/entrypoint.sh#L122 but during db_boostrapwe do connect using `DB_NAME=template1`:

https://github.com/astronomer/db-bootstrapper/blob/master/main.py#L68

but we want:
```
* = host=astronomer-postgresql.astronomer.svc.cluster.local port=5432 user=postgres
```

Reply:
@danielhoherd best i can do here just remove to not confuse.

Reply:
it's fine as it is. i didn't expect it to cause a problem.